apiservers/engine/CLI-COMPATIBILITY.md

```diff
@@ -0,0 +1,274 @@
+# Docker CLI Compatibility
+VIC attempts to be compatible with Docker client (CLI); however, there are some differences.  The CLI differences are documented below, alphabetically.  Note, the operations listed as supported but differences does not mean incompatible.  It means most of these operations support either a smaller subset of the parameters or the request/response parameters have different meanings that apply to running in an ESX or vCenter environment.
+
+The list is organized into 3 separate tables.  The first list CLI operations that is supported in the first version of VIC.  The second list are operations that VIC may support in the future.
```

Comment:
The list is organized into 3 separate tables.  The first table lists CLI operations that are supported in the first version of VIC.  The second table lists operations that VIC may support in the future.